config/api.yml

```diff
@@ -3992,6 +3992,12 @@
         :identifier:
         - vm_show
         - sui_vm_details_view
+      - :name: parent_choices
+        :identifier:
+        - we_don't_have_one_yet
```

Comment:
yes, it cloud be `vm_parent_choices` it is sound good to you.
we need to add it to `miq_product_fetaures.yml` in core repo - it could be hidden feature

Reply:
it's also one that's API specific, right?

Reply:
yes, we will use it only in API and not in UI (not sure If I answered you question.)